tokio/src/runtime/mod.rs

```diff
@@ -280,23 +281,23 @@ pub struct Runtime {
     handle: Handle,
 
     /// Blocking pool handle, used to signal shutdown
-    blocking_pool: BlockingPool,
+    blocking_pool: Arc<BlockingPool>,
```

Comment:
I assume that this is to support `Runtime` being `Clone`. I don't think this should be necessary.

Reply:
Yes